### PR TITLE
Add docs for Android (Kotlin)

### PR DESCRIPTION
**Pull request by zeienko-vitalii**
Currently this PR is in draft until all details are clarified and articles are ready to be properly reviewed and avoid accidental merges

### CODE REVIEW

File: docs/clients/server-side.md [hunk 1]
```diff
@@ -30,6 +30,11 @@ https://github.com/Flagsmith/flagsmith-python-client
 
 https://github.com/Flagsmith/flagsmith-java-client
 
+</TabItem>
+<TabItem value="kotlin" label="Kotlin">
```
Comment (matthewelwell) [inline docs/clients/server-side.md:34]: I think we should advertise kotlin for server side applications. It doesn't behave like any of our other server side clients. We would like to encourage people to just add the Java client as a dependency for kotlin server applications. 

What do you think @dabeeeenster ?

**Comment by dabeeeenster** [comment]
LGTM

**Review by dabeeeenster — APPROVED** [review]

File: docs/clients/client-side/android.md [hunk 2]
```diff
@@ -5,6 +5,147 @@ description: Manage your Feature Flags and Remote Config in your Android applica
...
+//flagsmith
+implementation 'com.github.Flagsmith:flagsmith-kotlin-android-client:1.0.1'
+```
+
+You should be able to find the latest version in the
+[releases section](https://github.com/Flagsmith/flagsmith-kotlin-android-client/releases) of the GitHub repository.
+
+### Gradle - Project
+
+In the new Gradle version 7+ update your `settings.gradle` file to include JitPack if you haven't already
```
[the remark this answers is not in the document]
Comment (gazreese) [reply docs/clients/client-side/android.md:27]: Hi @matthewelwell I went with JitPack in the end and appears to be working fine

**Comment by gazreese** [comment]
This should be all sorted now @dabeeeenster @matthewelwell

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**